src/stylesheets/base/_typography.scss

```diff
@@ -1,6 +1,6 @@
 body {
   color: $florence-gray;
-  font-family: 'geomanist';
+  font-family: $main-font, sans-serif;
```

Comment:
I'd move `sans-serif` in your `$main-font` variable (in _vars.scss), e.g. `$main-font: geomanist, sans-serif;`. Also, you won't need quotes around font names unless they contain whitespace or weird characters.

Reply:
Disregard. I'll move this comment to PR #3.